### DISCUSSION TITLE
Port Forwarding Deluge

### DISCUSSION THREAD

**Discussion by AdamF100001** [MichaIng/DietPi#6622] [Q&A]
Hi 

What is the default TCP / UCP port, I need to open, on my router so the torrent client connection will be open?

Thanks

**Comment by MichaIng** [comment]
For connections from remote peers for downloading/seeding, port 6882 is needed, I think generally both, UDP and TCP. If you want to connect to the web interface from remote browsers, TCP port 8112 is needed. But for security reasons, better to open it only via proxy and HTTPS, if needed at all.

**Reply by AdamF100001** [reply]
So, just to confirm, I can currently download torrents, but seedling sometimes just stops with tracker errors.

Will port forwarding port 6882 solve this issue?

Thanks

**Reply by MichaIng** [reply]
That makes sense, as a peer and tracker are not able to establish incoming connections with you, if the port is not forwarded.
